### PR TITLE
WIZZIE-2155 Update prozzie doc to latest changes

### PR DESCRIPTION
**Pull request by Chorro**

### CODE REVIEW

File: docs/installation/Installation.md [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 
-## Base linux installation
+## Base Linux installation
 
 ### Getting installation script
 #### Automagical installation
 
 Prozzie will be downloaded & installed if you execute the next command in a
-linux terminal:
+linux terminal as root user (not sudo command):
```
Comment (eugpermar) [inline docs/installation/Installation.md]: You can use `sudo bash -c ...` now

File: docs/protocols/mqtt.md [hunk 2]
```diff
@@ -4,17 +4,17 @@
 # MQTT
 
 ## Interactive script
-You can execute `setups/mqtt_setup.sh` to interactively configure one mqtt kafka
+You can run `prozzie config setup mqtt` to interactively configure one mqtt kafka
 connector. You will be asked for the next variables, that can't have any
 default:
```
Comment (eugpermar) [inline docs/protocols/mqtt.md]: Environment variables should also be warned here.

File: docs/protocols/snmp.md [hunk 3]
```diff
@@ -15,5 +15,5 @@ MONITOR_SENSORS_ARRAY='{"sensor_id":1,"timeout":2000,"sensor_name": "my-sensor",
 
 ## SNMP traps
 To listen for snmp in a specific port you have to use the `MONITOR_TRAPS_PORT`
-variable that will be asked in `monitor_setup.sh`. You can leave it blank for
+variable that will be asked in interactive mode of `prozzie config wizard` command. You can leave it blank for
```
Comment (eugpermar) [inline docs/protocols/snmp.md]: and setup monitor

**Comment by eugpermar** [comment]
I think that the commit message has to be "close #bla #bla #bla", and can't be separated by commas
This PR also close #63

File: docs/protocols/snmp.md (hunk 3 above)
Comment (Chorro) [reply docs/protocols/snmp.md]: This part is outdated, the monitor port is defined in its compose yaml. I've fixed the monitor documentation 👍

File: docs/installation/Installation.md (hunk 1 above)
Comment (Chorro) [reply docs/installation/Installation.md]: Done 👍

File: docs/protocols/mqtt.md (hunk 2 above)
Comment (Chorro) [reply docs/protocols/mqtt.md]: Do you mean about hidden variables?

**Comment by Chorro** [comment]
Done 👍

**Pull request merged**